File: python/7 kyu/197.py
# https://www.codewars.com/kata/5951b409aea9beff3f0000c6/train/python

# Passed

from math import gcd

def final_attack_value(x, monster_list):
    player_attack_points = x
    for monster_attack_points in monster_list:
        if player_attack_points >= monster_attack_points:
            player_attack_points += monster_attack_points
        else:
            player_attack_points += gcd(player_attack_points, monster_attack_points)
            
    return player_attack_points

output = final_attack_value(50,[50,105,200])
print(output)

File: nodejs/7 kyu/197.js
// https://www.codewars.com/kata/511f11d355fe575d2c000001/train/javascript

// Passed

function twoOldestAges(ages) {
  let sortedAges = ages.sort((a, b) => a - b);
  let agesToReturn = sortedAges.slice(sortedAges.length - 2);
  return agesToReturn;
}

const output = twoOldestAges([1, 5, 87, 45, 8, 8]);
console.log(output);